module_intros/diffraction.StaticStructureFactor.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        sfDebye.compute(frame, reset=False)
</pre><p>You can just pass the entire frame object, and freud will extract box/points from the GSD frame automatically. Then delete the lines defining box/points.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>